capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -198,6 +215,10 @@ class ClubPage extends Component {
 
               <div className='text-center'>
                 <div className='description'> {this.state.club.description} </div>
+                <div> Upcoming Meetings: </div>
+                  <CardDeck className='groups-list-container'>
+                    {this.state.meetings.map(m => <MeetingCard meeting={m} deleteMeeting={this.deleteMeeting} />)}
+                  </CardDeck>
```

Comment:
This is really nice and clean, good work!

Reply:
Thanks!